spynnaker/pyNN/models/neuron/implementations/neuron_impl_standard.py

```diff
@@ -45,18 +45,20 @@ class NeuronImplStandard(AbstractNeuronImpl):
         "__n_steps_per_timestep"
     ]
 
-    _RECORDABLES = ["v", "gsyn_exc", "gsyn_inh"]
+    _RECORDABLES = ["v", "gsyn_exc", "gsyn_inh", "rewiring"]
```

Comment:
This can be removed from here and instead treated more like packets per ts or spikes